src/content/en/showcase/2017/santa.md

```diff
@@ -0,0 +1,217 @@
+When it changes, it knows to resolve the `path` attribute of a scene, and use HTML imports to load it.
+
+1. The element `<boatload-scene>` is unknown and is treated as a HTMLUnknownElement with some additional attributes
+2. The `<lazy-pages>` element chooses the scene, loading the HTML import `scenes/boatload/boatload-scene_en.html`—this contains the Polymer element and its dependant elements
+3. The friendly preloader is shown
+4. Once the HTML import is loaded and executed, `<boatload-scene>` is transparently upgraded to a real Polymer element, full of holiday cheer 🎄🎉
+
+This approach has its challenges—for example, we don't want to include duplicate web components.
+If two scenes use a common element, e.g., `paper-button`, we strip it out as part of our build process and instead include it in Santa's shared code.
+
+## Offline Design
+
+Santa Tracker is already neatly segmented into scenes thanks to Polymer and `lazy-pages`; plus each scene has its own directory.
+We designed Santa Tracker's Service Worker, the core piece that enables offline which runs on a user's browser, to be aware of the shared code vs 'scene' distinction.
+
+What's the theory behind Service Worker? When a user on a supported browser loads your site, the frontend HTML can request that the Service Worker (SW) is installed.
+For Santa Tracker, the SW lives at [`/sw.js`](https://santatracker.google.com/sw.js).
+This then runs an 'install' event that will precache all of Santa's shared code, so nothing needs to be fetched at runtime.
+
+<figure>
+  <img src="/web/showcase/2017/images/santa/sw-flow.png"  />
+</figure>
+
+The Service Worker, once installed, is able to intercept all HTTP requests.
+For Santa Tracker, the simplified decision flow looks like-
+
+1. Is the request already cached?
+  * Great! Return the cached response.
+2. Does the request being made match a scene directory, like "scenes/boatload/boatload-scene_en.html"?
+  * Perform a network request, and store the result in the cache before returning it to the user.
+3. Otherwise, perform a regular network request.
+
+Our flow and 'install' event allows Santa Tracker to load, even while a user is offline.
+However, only the scenes a user has previously loaded will be available—perfect for replaying that one game and beating your high score.
+
+Keen observers may note that our caching strategy doesn't allow for _changes_ in content.
+Once a file is cached in a user's browser, it will never be changed.
+More on that later!
+
+### We'll Do It Live
+
+As we mentioned, Santa's elves also hard at work throughout December, and they often have to release new updates throughout the month.
+When a release of Santa Tracker is built, it is given a unique label—e.g., `v20161204112055`, the timestamp of the release (11:20:55 on the 4th of December, 2016).
+
+For this labelled release, we generate a MD5 hash of every file and store this in our 'cache manifest'.
```

Comment:
Did you define cache manifest somewhere? Can you, preferably at the first mention?